tools/modules/system/manage_dtoverlays.sh

```
@@ -18,9 +18,10 @@ function manage_dtoverlays () {
 	# check if user agree to enter this area
 	local changes="false"
 	local overlayconf="/boot/armbianEnv.txt"
-	local overlaydir="$(find /boot/dtb/ -name overlay -and -type d)"
+	# Raspberry Pi has different name
+	local overlaydir="$(find /boot/dtb/ -type d \( -name overlay -o -name overlays \))"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)